# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1318238832 -3600
#      Mon Oct 10 10:27:12 2011 +0100
# Node ID cbdefe0ec5143034c9563a73a583ce9c86289713
# Parent  ad9b78544a222936f7fd6b4515517be0bb08e1ca
Add __object__ property to uimenu.

* graphics.h.in (uimenu::properties::__object__): New property.

diff --git a/src/graphics.h.in b/src/graphics.h.in
--- a/src/graphics.h.in
+++ b/src/graphics.h.in
@@ -4568,16 +4568,17 @@ public:
       {
         base_properties::adopt (h);
       }
 
     // See the genprops.awk script for an explanation of the
     // properties declarations.
 
     BEGIN_PROPERTIES (uimenu)
+      any_property __object__ , Matrix ()
       string_property accelerator , ""
       callback_property callback , Matrix()
       bool_property checked , "off"
       bool_property enable , "on"
       color_property foregroundcolor , color_values (0, 0, 0)
       string_property label , ""
       double_property position , 9
       bool_property separator , "off"
